localprod/setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+[[ -f ./openslides ]] ||
+  wget https://github.com/OpenSlides/openslides-manage-service/releases/download/latest/openslides
+
+[[ -x ./openslides ]] ||
+  chmod +x ./openslides
+
+./openslides setup .
```

Comment:
This can be a oneliner here `./openslides setup --config config.yml .`

Reply:
For first time setup yes.
But I want to be able to re-run `setup.sh` after e.g. changing the config file. So for that to work I think this should stay like this

Reply:
Ok for me